modules/loadbalancer/templates/nginx_balance.conf.erb

```diff
@@ -4,7 +4,13 @@ upstream <%= @vhost %>-upstream {
 <%- end -%>
 }
 
-<%- if @https_only -%>
+<%- if @error_on_http -%>
+server {
+  server_name <%= @vhost_real %> <%= @aliases.join(" ") unless @aliases.empty? %>;
+  listen 80;
+  return 426 "Request cannot be completed using a plaintext HTTP connection\n";
```

Comment:
This could specify that HTTPS must be used (seems obvious to us but might not be immediately obvious to everyone).